Source/interactivity_manager_impl.cpp

```diff
@@ -1590,17 +1613,22 @@ void interactivity_manager_impl::process_method(const web::json::value& methodJs
             else
             {
                 LOGS_INFO << "Unexpected or unsupported RPC call: " << methodJson.at(RPC_METHOD).as_string();
+                messageWasProcessed = false;
             }
         }
         else
         {
             LOGS_INFO << "Method RPC call had no method name";
+            messageWasProcessed = false;
         }
     }
     catch (std::exception e)
     {
         LOGS_ERROR << "Failed to process method RPC call";
+        messageWasProcessed = false;

```

Comment:
>messageWasProcessed = false; [](start = 8, length = 28)

again, this seems like a true error state, and should not be passed back to the caller as a "valid" message. The exception generally thrown here is a json parsing exception, which we should not be handing back. Unless you all are allowing non-json messages. :)